mapie/utils.py

```diff
@@ -1373,3 +1373,37 @@ def check_n_samples(
              " int in the range [1, inf)"
              )
     return int(n_samples)
+
+
+def check_predict_params(
+    predict_params_used_in_fit: bool,
+    predict_params: dict,
+    cv: Optional[Union[int, str, BaseCrossValidator]] = None
+) -> None:
+    """
+    Check that if predict_params is used in the predict method,
+    it is also used in the fit method. Otherwise, raise an error."
```

Comment:
```suggestion
    Check that if predict_params is used in the predict method,
    it is also used in the fit method. Otherwise, raise an error.
```